packages/mobile/src/screens/profile-screen/ProfileHeader/ProfileInfoTiles.tsx

```diff
@@ -292,7 +292,13 @@ const RelatedArtistsTile = ({ userId }: { userId: number }) => {
       content={
         <UserListWithCount
           users={relatedArtists}
-          count={isLoading ? MAX_CARD_PROFILE_PICTURES : relatedArtists.length}
+          count={
+            isLoading
+              ? MAX_CARD_PROFILE_PICTURES
+              : relatedArtists.length < MAX_CARD_PROFILE_PICTURES
+                ? relatedArtists.length
+                : 100
+          }
```

Comment:
can we extract this into its own var and add a variable to 100 so its usage is clear?

Reply:
ya ofc